verilog/tools/lint/lint_tool_test.sh

```diff
@@ -359,8 +371,8 @@ check_diff() {
 
   # Show invisible characters
   local invisible_chars_expr_=':join;N;$!b join; s/ /·/g;s/'$'\t''/├───/g;s/'$'\r''/⁋/g;s/\n/¶\n/g'
```

Comment:
Do you still need this `invisible_chars_expr_` now ?

Reply:
Line removed